tests/messages_tests/test_cookie.py

```diff
@@ -116,15 +120,42 @@ def test_max_cookie_length(self):
         # size which will fit 4 messages into the cookie, but not 5.
         # See also FallbackTest.test_session_fallback
         msg_size = int((CookieStorage.max_cookie_size - 54) / 4.5 - 37)
+        first_msg = None
+        random.seed(42)
         for i in range(5):
-            storage.add(constants.INFO, str(i) * msg_size)
+            s = str(i) + ''.join(random.choice(string.ascii_letters) for _ in range(msg_size - 1))
+            storage.add(constants.INFO, s)
+            if i == 0:
+                first_msg = s
         unstored_messages = storage.update(response)
 
         cookie_storing = self.stored_messages_count(storage, response)
         self.assertEqual(cookie_storing, 4)
 
         self.assertEqual(len(unstored_messages), 1)
-        self.assertEqual(unstored_messages[0].message, '0' * msg_size)
+        self.assertEqual(unstored_messages[0].message, first_msg)
+
+    def test_message_rfc6265_compliant(self):
+        non_compliant_chars = ('\\', ',', ';', '"')
+        messages = ['\\te,st', ';m"e', '\u2019']
+        storage = self.get_storage()
+        encoded = storage._encode(messages)
+
+        for illegal in non_compliant_chars:
+            self.assertEqual(encoded.find(illegal), -1)
+
+    def test_messages_store_various_charsets(self):
```

Comment:
Do we need this test? I am usually not against adding new tests; but I do not think this test adds much value. Or differently put: Since we run this simply through `signing.sign_object` the test would belong there (if at all) -- I'd just drop it.

Reply:
(And round-tripping of the messages is already tested in other tests)

Reply:
Although round-tripping is tested elsewhere, this test adds tests for characters from other charsets that I didn't see elsewhere. I thought it prudent to include it here since one of the failures that can occur, if storage is implemented differently, is for cookie storage not to recognise a latin-1 quote character that probably comes from WSGI. Please let me know if you still think it's better to remove it. Or if I should make the reason for the test more explicit.

Reply:
Ah - sorry, Florian. I feel stupid. Of course there are existing tests that picked up the need to handle different charsets. I will plan to remove the test tomorrow. Happy New Year :tada: